wrappers/wrapper.rb

```diff
@@ -64,19 +62,6 @@ def assert_vehicles_no_alternative_skills(vrp)
       vrp.vehicles.none?{ |vehicle| vehicle.skills.size > 1 }
     end
 
-    def assert_no_direct_shipments(vrp)
```

Comment:
I think this should still be checked.

Reply:
A direct shipment is converted into a sequence. Vroom is only candidate if the relation is a shipment. Then this check is no more needed

Reply:
I was thinking of. If the user uses the shorcut `api:shipment` and send us a `direct` shipment we will refuse the vroom solver with `sequence` relation explanation :/ Not ideal but at the same time not so important either.